app/src/main/java/net/squanchy/favorites/view/FavoritesItem.kt

```diff
@@ -0,0 +1,34 @@
+package net.squanchy.favorites.view
+
+import net.squanchy.schedule.domain.view.Event
+import org.joda.time.LocalDate
+
+sealed class FavoritesItem {
+
+    abstract val id: Long
+
+    abstract val type: Type
+
+    data class Header(val date: LocalDate) : FavoritesItem() {
+
+        override val id: Long
+            get() = date.toDateTimeAtCurrentTime().millis // We're relatively sure this won't clash with event IDs
+
+        override val type: Type
+            get() = Type.HEADER
```

Comment:
any reason for the getter? Why not simply `override val type = Type.Header`?

Reply:
Just whatever the IDE put when I did override... fixed f17973d122a060b5d61f295b71de5244eb98c102